syz-verifier/verifier.go

```diff
@@ -142,6 +142,7 @@ func (vrf *Verifier) TestProgram(prog *prog.Prog) (result []*ExecResult) {
 
 	defer vrf.stats.TotalProgs.Inc()
```

Comment:
No need for defer, it's not atomic with other increments anyway.

Reply:
Didn't get your message.
I wanted to inc this counter once analysis is done, independently of the result (was it positive, negative or failed).
Used defer because wanted to inc after the analysis procedure, not before or in the middle.